Calculator/src/main/java/com/microsoft/device/display/samples/calculator/models/Calculations.kt

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.calculator.models
```

Comment:
I think you may reconsider how to use the math operations below. In all functions there is not an input and output, it takes its input and writes its output in an external model (think about this as an external variable). All these functions cannot be unit tested since you can not provide specific input and check the result as an output. This is easier to see when implementing tests, but basically, you cannot test, let say, `add` function, because you don't have a way to provide the set of numbers you want to use in the add operation and check what the function returns as its result, because the function is nor returning anything.

The correct way would be:
```
fun add(param1: Int, param2: int): Int {
return param1 + param2 
}
```
Then when testing you could do:
```
@Test
fun should_my_add_function_do_the_math_correctly() {
    val expected = 3
    val result = add (1, 2)
    assertThat (result, is(expected))
}
As a good rule to follow, try always to avoid global objects that can be called along the whole app and modified everywhere since it will be difficult/impossible to test + there will be many side effects that you cannot control and give you problems.